cnf-certification-test/certification/suite.go

```diff
@@ -150,3 +151,28 @@ func testHelmCertified(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("%d helms chart are not certified.", len(failedHelmCharts)))
 	}
 }
+func testContainerCertificationStatusByDigest(env *provider.TestEnvironment) {
+	containersToQuery := certtool.GetContainersToQuery(env)
+	testhelper.SkipIfEmptyAny(ginkgo.Skip, containersToQuery)
+	ginkgo.By(fmt.Sprintf("Getting certification status. Number of containers to check: %d", len(containersToQuery)))
+	failedContainers := []configuration.ContainerImageIdentifier{}
+	allContainersToQueryEmpty := true
+	for c := range containersToQuery {
+		if c.Name == "" || c.Repository == "" {
+			tnf.ClaimFilePrintf("Container name = \"%s\" or repository = \"%s\" is missing, skipping this container to query", c.Name, c.Repository)
+			continue
+		}
+		allContainersToQueryEmpty = false
+		justDigest := true
+		if !testContainerCertification(c, justDigest) {
```

Comment:
you can test if the digest is empty here and fail, this way. This way, there is no need for the justDigest param